lib/components/app/batch-routing-panel.js

```diff
@@ -9,6 +9,7 @@ import LocationField from '../form/connected-location-field'
 import NarrativeItineraries from '../narrative/narrative-itineraries'
 import { getActiveSearch, getShowUserSettings } from '../../util/state'
 import ViewerContainer from '../viewers/viewer-container'
+import switchButton from '../form/switch-button'
```

Comment:
Component names should generally always be capitalized:
```suggestion
import SwitchButton from '../form/switch-button'
```

Reply:
Thanks for the feedback. 

I am not sure what happened there. It was working on my machine.  When I loaded it up today it wasn't working...

I changed the code a little, made your change and cleared my cache and it is now working.